data/strings/strings.txt

```diff
@@ -7499,6 +7499,51 @@
     zh-Hans = 以八字动作移动手机，校准罗盘，并在地图上固定箭头方向。
     zh-Hant = 以 8 字形移動手機以校準指南針並固定地圖上的箭頭方向
 
+  [long_tap_toast]
+    comment = Toast text when user hides UI with a long tap anywhere on the map
+    tags = android,ios
+    en = Long-tap on the map again to see the interface
```

Comment:
Actually **any** tap on the map brings the interface back.
Should we change it?

Reply:
A tap on the map should select POI and show a place page. It should not show back the interface in the "browse map" mode.

CC @strump